src/components/__tests__/ContactLookupField.test.tsx

```diff
@@ -132,8 +131,9 @@ describe('ContactLookupField', () => {
     const { container } = getComponent(getDefaultProps(), contacts, output);
     const input = container.querySelector('input');
     fireEvent.click(input!);
+    input!.focus();
```

Comment:
Why are these needed? They are added in multiple places.

Reply:
Now with testing library when you want to type in a input you need to focus first, see https://testing-library.com/docs/guide-events/